themes/navy/languages/fr.yml

```diff
@@ -0,0 +1,69 @@
+menu:
+  docs: Docs
+  api: API
+  news: Nouvelles
+  plugins: Plugins
+  themes: Thèmes
+  search: Chercher
+
+index:
+  get_started: Commençer
```

Comment:
s/Commençer/Commencer/

Reply:
sorry I don't understand

Reply:
Sorry, I used a "grep" syntax. 

I suggested to fix the typo by using the term `Commencer` (the `ç` is replaced by `c`).